examples/README.md

```diff
@@ -18,3 +18,21 @@ Simply run `make` in this directory.
 % make clean
 % make
 ```
+
+Examples Docker image
+---------------------
+You can experiment with [http3-client](http3-client.rs), [http3-server](http3-server.rs), [client](client.rs) and [server](server.rs) using Docker.
+
+The Examples [Dockerfile](Dockerfile) builds a Debian image.
+
+To build:
+
+```
+docker build -t cloudflare-quiche .
+```
+
+To run http3-client once the image is built:
+
+```
+docker run --entrypoint '/bin/bash' cloudflare-quiche -c "http3-client https://cloudflare-quic.com"
```

Comment:
Do we need the `--entrypoint` parameter? Ideally we'd just do:
```
docker run -it cloudflare-quiche http3-client https://cloudflare-quic.com
```
or something like that.

Reply:
Default entrypoint is now http3-client, so all user needs to provide is a URL like your example. H@ck3r$ can override this if they really want to.